app/javascript/gobierto_data/webapp/components/commons/Queries.vue

```diff
@@ -89,7 +89,7 @@
       <div class="pure-u-1-2 border-color-queries">
         <p class="gobierto-data-summary-queries-sql-code">
           <transition name="fade">
-            <span v-if="sqlCode"> {{ sqlCode }}</span>
+            <textarea ref="querySnippet" />
```

Comment:
Missing `v-if`

Reply:
Well, this is a bit tricky, now we can't use `v-if ` in `textarea.` Why? Because CodeMirror needs to hide the element to build your logic and structure, and if we put a `v-if ``CodeMirror` cannot initialize because `textarea` don't exist when the component is created. What's happen when `sqlCode` get a value? Vue show a default `textarea` without code and CodeMirror return an error

![Screenshot 2020-04-21 at 13 25 53](https://user-images.githubusercontent.com/2649175/79860849-b074bc80-83d3-11ea-842b-93a9ba2d48f9.png)

Reply:
Try then `v-show`. If not, remove the `transition`

Reply:
Same problem, `v-show` show/hide default `textarea`